### DISCUSSION TITLE
scroll in scrollview

### DISCUSSION THREAD

**Discussion by persiabudi28** [dotnet/maui#4459] [Q&A] [answered]
how to make both scroll, vertical and horizontal scrollview

**Comment by davidbuckleyni** [comment, marked as answer]
Its not a bug plz do ur research voting to close this. https://docs.microsoft.com/en-us/xamarin/xamarin-forms/user-interface/layouts/scrollview#scroll-programmatically

@PureWeen can u close this issue they not done their research

**Reply by persiabudi28** [reply]
please use this code for scrollview both orientation, you get scroll vertical and horizontall(both) not working, but you select one vertical or horizontal 

<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
             x:Class="scroll.MainPage"
             BackgroundColor="{DynamicResource SecondaryColor}">

    <ScrollView Orientation ="Both">
        <Grid RowSpacing="25" ColumnDefinitions="Auto" RowDefinitions="Auto"
              Padding="{OnPlatform iOS='30,60,30,30', Default='30'}">        

            <Image Grid.Row="0" Grid.Column="0"
                Source="dotnet_bot.png"
                SemanticProperties.Description="Cute dot net bot waving hi to you!"
                WidthRequest="900"
                HeightRequest="900"
                HorizontalOptions="Center" />
        </Grid>
    </ScrollView>
</ContentPage>
